### PR TITLE
fix(staking): Return the empty value instead of error when not found

### PR DESCRIPTION
**Pull request by calbera**

### CODE REVIEW

File: cosmos/precompile/staking/helpers.go [hunk 1]
```diff
@@ -235,7 +239,10 @@ func (c *Contract) validatorsHelper(ctx context.Context) ([]any, error) {
 	res, err := c.querier.Validators(ctx, &stakingtypes.QueryValidatorsRequest{
 		Status: stakingtypes.BondStatusBonded,
 	})
-	if err != nil {
+	if status.Code(err) == codes.NotFound {
```
Comment (itsdevbear) [inline cosmos/precompile/staking/helpers.go:242]: errors.Is(....)

File: cosmos/precompile/staking/helpers.go [hunk 2]
```diff
@@ -252,7 +259,10 @@ func (c *Contract) validatorHelper(ctx context.Context, valAddr string) ([]any,
 	res, err := c.querier.Validator(ctx, &stakingtypes.QueryValidatorRequest{
 		ValidatorAddr: valAddr,
 	})
-	if err != nil {
+	if status.Code(err) == codes.NotFound {
```
Comment (itsdevbear) [inline cosmos/precompile/staking/helpers.go:262]: errors.Is(....)

File: cosmos/precompile/staking/helpers.go [hunk 3]
```diff
@@ -270,7 +280,10 @@ func (c *Contract) delegatorValidatorsHelper(ctx context.Context, accAddr string
 	res, err := c.querier.DelegatorValidators(ctx, &stakingtypes.QueryDelegatorValidatorsRequest{
 		DelegatorAddr: accAddr,
 	})
-	if err != nil {
+	if status.Code(err) == codes.NotFound {
```
Comment (itsdevbear) [inline cosmos/precompile/staking/helpers.go:283]: errors.Is(....)

Comment (calbera) [reply cosmos/precompile/staking/helpers.go:283]: doesn;t work

**Comment by itsdevbear** [comment]
ahh unlucky

**Review by itsdevbear — APPROVED** [review]
utACK

**Pull request merged**